rfc8446.xml

```diff
@@ -604,9 +630,9 @@ opaque.</t>
 <section anchor="numbers" title="Numbers">
 
 <t>The basic numeric data type is an unsigned byte (uint8). All larger numeric
-data types are formed from fixed-length series of bytes concatenated as
-described in <xref target="basic-block-size"/> and are also unsigned. The following numeric
-types are predefined.</t>
+data types are formed from fixed-length serieses of bytes concatenated as
```

Comment:
Are you sure? "Serieses" might be technically acceptable, but "series" as a plural is far more common (and this exact sentence with "series" instead of "serieses" is present in every TLS version's RFC since SSL 3.0).